data-collection/src/test/java/integration/realprojects/ApacheCommonsCSVIntegrationTest.java

```diff
@@ -120,7 +120,7 @@ public void stable_CSVFormat() {
 
         // in refactorings_CSVFormat, we see that there are 82 refactorings in total.
         // after this commit, there was just one more refactoring. Thus, 81 refactorings
-        Assert.assertEquals(194, stableCommitListLevel2.get(0).getProcessMetrics().refactoringsInvolved);
+        Assert.assertEquals(195, stableCommitListLevel2.get(0).getProcessMetrics().refactoringsInvolved);
```

Comment:
I see that you are counting the rename you find from JGit as a refactoring. Am I right?

Are you sure you are not double counting it? Moreover, isn't it better not to count it, i.e., only counts when RefactoringMiner identifies it? RefactoringMiner uses a bit more semantic to decide whether that was a rename or not, so only looking at file renames might end up in cases we don't fully understand.

Reply:
I manually validated the new cases and they are correct. We have some tests in the Toyprojects covering these cases. Therefore, I am quite confident, that we are not double counting.